test/cli.test.mjs

```diff
@@ -0,0 +1,62 @@
+import chai from "chai";
+import shell from "shelljs";
```

Comment:
Using the shelljs module to run the cli command and respond to the question it asks.  This module will also be useful in getting this all running in windows

Reply:
Oh fun!